.travis.yml

```diff
@@ -22,6 +22,13 @@ jobs:
     after_success:
     - yarn add -D semantic-release@11.0.1
     - yarn semantic-release
+  - stage: danger
+    node_js: 8
+    os: linux
+    script: 
+      - yarn add -D danger@^2
```

Comment:
`npx danger@^2`?

Reply:
@cpojer where is my `yarn exec danger` ;)

Reply:
Agree though @SimenB

Reply:
https://github.com/yarnpkg/yarn/issues/3937